src/viam/components/board/board.py

```diff
@@ -38,11 +39,28 @@ class Analog:
         name: str
         """The name of the analog pin"""
 
+        @dataclass
+        class Value:
```

Comment:
You could probably make this a `TypeAlias` if all the attributes are the same

```
if sys.version_info >= (3, 10):
    from typing import TypeAlias
else:
    from typing_extensions import TypeAlias

class Board:
    class Analog:
        Value: "TypeAlias" = ReadAnalogReaderResponse
```

Similar to how it's done in [Vision](https://github.com/viamrobotics/viam-python-sdk/pull/617/files#diff-a7311669680bf0703ece9bd2c4887bb9bf823cd40ffd8c4154d56c4f38733e15R58)

Reply:
This way, we don't have to keep `Value` in sync with the response type should any fields be added/removed

Reply:
Oh dang, that's _super_ clean! I had no idea that was a thing. Thanks!